client/src/components/Admin/AccountAutocomplete.tsx

```diff
@@ -1,15 +1,30 @@
 import { Autocomplete, TextField } from "@mui/material";
 import { useAccounts } from "../../hooks/useAccounts";
 
+interface Account {
+  id: string | number;
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+interface AccountAutocompleteProps {
+  accountId: string | number | null;
+  setAccount?: (account: Account | null) => void;
+  setAccountId?: (id: string | number | null) => void;
+  label?: string;
+  name?: string;
+}
+
 export default function AccountAutocomplete({
   accountId,
   setAccount,
   setAccountId,
   label,
-}) {
+  name,
+}: AccountAutocompleteProps) {
   const { data: accounts } = useAccounts();
 
-  const handleChange = (event, value) => {
+  const handleChange = (_event: unknown, value: Account | null) => {
```

Comment:
I think this event is `React.SyntheticEvent` but if we're not using it, we prob shouldn't pass it in 🤔